docs/core/tutorials/top-level-templates.md

```diff
@@ -38,11 +38,11 @@ These two forms represent the same program. Both are valid with C# 10.0. When yo
 
 ## Use the new program style
 
-The features that make the new program simpler are top-level statements, global usings, and implicit usings.
+The features that make the new program simpler are *top-level statements* and *global using directives*.
```

Comment:
We should keep "implicit using directives" here. (I haven't written the two articles I'll reference here yet, but there is a difference. "global usings" are the statements you add. "implicit using directives" are a set of global usings that MSBuild will create using a source generator).

```suggestion
The features that make the new program simpler are *top-level statements*, *global using directives*, and *implicit using directives*.
```